### PR TITLE
Update the URL for the requests package

### PR DESCRIPTION
**Pull request by simon04**
Change the url from docs.python-requests.org to requests.readthedocs.io

Automerge-Triggered-By: @Mariatta

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@simon04

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by brandtbucher — APPROVED** [review]
Thanks for your time @simon04, and welcome to CPython! :sunglasses:

Looks good to me.

File: Doc/library/http.client.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ HTTPS protocols.  It is normally not used directly --- the module
 
 .. seealso::
 
-    The `Requests package <http://docs.python-requests.org/>`_
+    The `Requests package <https://requests.kennethreitz.org/>`_
```
Comment (Mariatta) [inline Doc/library/http.client.rst]:
```suggestion
    The `Requests package <https://requests.readthedocs.io/en/master/>`_
```

**Review by Mariatta — CHANGES_REQUESTED** [review]
Thanks for the fix, however the right url is https://requests.readthedocs.io/en/master/

**Comment by simon04** [comment]
I have made the requested changes; please review again

**Review by Mariatta — APPROVED** [review]

**Comment by brandtbucher** [comment]
Congrats on your first CPython contribution @simon04! :champagne:

Looking forward to seeing more from you in the future.

**Pull request merged**